Automattic/simplenote-android
# Select next note after trashing/restoring a note in tablet landscape

Fixes #1021 

### Fix
This PR adds implementation of the bellow suggestions:
- Select the note below a recently trashed note.
- Select the note below a recently restored note in trash view.

The goal is to offer feature parity with other platforms.

![trash_select](https://user-images.githubusercontent.com/1657201/102495624-9aa8bb80-4076-11eb-8570-b66b35f8e160.gif)


### Test
#### Selecting the note below the trashed note:
1. Open the notes list using a tablet in landscape mode
2. Open a note that's not the latest in the list.
3. Trash the note.
4. Make sure that the note below is selected.

#### Selecting the note above the trashed note:
1. Open the notes list using a tablet in landscape mode
2. Open a latest note in the list.
3. Trash the note.
4. Make sure that the note above is selected.

#### Trashing the latest note:
1. Open the notes list using a tablet in landscape mode
2. Trash all note except one, and open it.
3. Trash the note.
4. Make sure that the note placeholder is shown.

**Important** Please try to run the tests on a Chromebook too, as I don't have one to test.

### Review
Only one developer is required to review these changes, but anyone can perform the review.

### Release
These changes do not require release notes.
